lab2.js

```diff
@@ -74,39 +98,52 @@ assert(blob.hoursToOoze(1000, 1) === hoursSpentInDowington,
 // PROBLEM 2: Universal Translator. 20 points
 //*********************************************************
 
-var hello = {
-  klingon: 'nuqneH',  // home planet is Qo'noS
-  romulan: 'Jolan\'tru', // home planet is Romulus
-  'federation standard': 'hello' // home planet is Earth
-};
+// var hello = {
+//   klingon: 'nuqneH',  // home planet is Qo'noS
+//   romulan: 'Jolan\'tru', // home planet is Romulus
+//   'federation standard': 'hello' // home planet is Earth
+// };
 
 // TODO: define a constructor that creates objects to represent
 // sentient beings. They have a home planet, a language that they
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
-  // TODO: specify a home planet and a language
-  // you'll need to add parameters to this constructor
+function SentientBeing(planet, language) {
+  this.planet = planet;
+  this.language = language;
 }
 
 // sb is a SentientBeing object
-function sayHello (sb) {
+SentientBeing.prototype.sayHello = function(sb) {
     // TODO: say hello prints out (console.log's) hello in the
     // language of the speaker, but returns it in the language
     // of the listener (the sb parameter above).
     // use the 'hello' object at the beginning of this exercise
     // to do the translating
-
+    console.log(this.language);
+    return sb.language;
```

Comment:
close. You're passing in the word "hello" (or it's equivalent) to the constructor, instead of the language. You can then use the language, the hello object, and some bracket ntation, along with what you have here, to console log/return the word.